docs/data-ai/quickstart/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+linkTitle: "Quickstart"
+title: "Quickstart"
+weight: 1
+layout: "docs"
+type: "docs"
+open_on_desktop: true
+header_only: true
+---
```

Comment:
Please make this an empty parent, unless the plage itself communcates something of value that isn't just "there's three steps in these subpages"